src/main/kotlin/app/matty/api/auth/TokenService.kt

```diff
@@ -0,0 +1,54 @@
+package app.matty.api.auth
+
+import app.matty.api.auth.data.RefreshTokenRepository
+import app.matty.api.auth.data.TokenPair
+import app.matty.api.auth.exc.InvalidTokenException
+import app.matty.api.auth.exc.TokenServiceException
+import app.matty.api.auth.exc.UnidentifiedRefreshToken
+import app.matty.api.user.data.User
+import app.matty.api.user.data.UserRepository
+import org.springframework.stereotype.Service
+
+@Service
+class TokenService(
+    private val tokenGenerator: TokenGenerator,
+    private val tokenDataExtractor: TokenDataExtractor,
+    private val refreshTokenRepository: RefreshTokenRepository,
+    private val userRepository: UserRepository
+) {
+    fun emmitTokens(user: User): TokenPair {
```

Comment:
```suggestion
    fun emitTokens(user: User): TokenPair {
```

Reply:
Thanks for noticing 🙂 I'll fix this typo